sellercenter-attributes/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.gfg.sellercenter.attributes</groupId>
+    <artifactId>sellercenter-attributes</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>sellercenter.attributes</name>
+    <url>https://github.com/GFG/java-packages</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <java.version>11</java.version>
+        <jackson.version>2.10.3</jackson.version>
+        <jackson-bom.version>${jackson.version}</jackson-bom.version>
```

Comment:
I see no jackson among packages, so I guess it can be also removed. However, you know my point of view about it and I suggest to use it for all serializing/deserializing stuff.